pgmq-extension/sql/pgmq--1.4.1--1.4.2.sql

```diff
@@ -1,3 +1,14 @@
+CREATE OR REPLACE FUNCTION pgmq.format_table_name(queue_name text, prefix text)
+RETURNS TEXT AS $$
+BEGIN
+    IF queue_name ~ '\$|;|--|''|\s'
+    THEN
+        RAISE EXCEPTION 'queue name contains invalid characters: $, ;, --, or \''';
```

Comment:
Does not mention that `\s` is disallowed.

Reply:
going to allow spaces